### PR TITLE
Return ResponseWrapper objects instead of dictionaries

### PR DESCRIPTION
**Pull request by rsaestrela**

### CODE REVIEW

**Review by Armando-Ibarra — APPROVED** [review]

File: checkout_sdk/checkout_response.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+class ResponseWrapper(object):
```
Comment (michaeljaszczuk) [inline checkout_sdk/checkout_response.py:1]: Skd support pytahon>3 so no need to inherit object.

File: checkout_sdk/checkout_response.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+    def _wrap(self, value):
+        if self._is_collection(value):
+            return type(value)([self._wrap(v) for v in value])
+        else:
+            return ResponseWrapper(None, value) if isinstance(value, dict) else value
+
+    @staticmethod
+    def _is_collection(value):
+        return isinstance(value, (tuple, list, set, frozenset))
```
Comment (michaeljaszczuk) [inline checkout_sdk/checkout_response.py:21]: Maybe it's good to create global utils file for those kind of methods and keep them there, wyt?

**Review by michaeljaszczuk — APPROVED** [review]
I love that concept!

File: checkout_sdk/checkout_response.py (hunk 1 above)
Comment (rsaestrela) [reply checkout_sdk/checkout_response.py:1]: Nice catch. Will remove it.

File: checkout_sdk/checkout_response.py (hunk 2 above)
Comment (rsaestrela) [reply checkout_sdk/checkout_response.py:21]: Sounds good. We can do that later, because atm it's not being reused in any other place.

**Pull request merged**